lib/test_prof/event_prof/minitest.rb

```diff
@@ -37,7 +38,7 @@ def after_test(*); end
     def report
       @profiler.group_finished(@current_group)
       result = @formatter.prepare_results
-
+      puts "\n"
```

Comment:
Bad idea. We should output everything using `log`, 'cause (in theory) it can write to file.
Just prepend the `result` with `"\n"` (i.e. `log :info, "\n#{result}"`).

Reply:
Actually, it won't help with the concatenation issue, because we need new line before log message itself, but not before the results. But as I wrote below I agree that it was a dumb mistake to didn't keep in mind a possibility to write results to the file :)